File: Python/1000-1999/1159.py
def main():
    while True:
        x = int(input())

        if x == 0:
            break

        soma = 0
        cont = 5
        while cont > 0:
            if x%2 == 0:
                soma += x
                cont -= 1
            x += 1
        print(soma)

main()

File: C/1000-1999/1159.c
#include <stdio.h>

int main (void){
    int res, soma = 0;
    scanf("%d", &res);
    while (res != 0){
        soma = 0;
        for (int c = res; c < res+10; c++){
            if (c%2 == 0)
                soma += c;
        }
        printf("%d\n", soma);
        scanf("%d", &res);
    }

    return 0;
}
